File: rental_manager_server/src/main/kotlin/pl/monochrome/manager/model/database/Person.kt
package pl.monochrome.manager.model.database

import javax.persistence.*

@Entity
@Table(name = "people")
data class Person(

        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        val id: Int,

        @Column
        val name: String,

        @Column
        val surname: String,

        @Column
        val phone: String?,

        @Column
        val mail: String?

)

File: rental_manager_app/lib/model/person.dart

class Person {
  int id;
  String name;
  String surname;
  String phone;
  String mail;

  Person({this.id, this.name, this.surname, this.phone, this.mail});

  Person.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    name = json['name'];
    surname = json['surname'];
    phone = json['phone'];
    mail = json['mail'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['name'] = this.name;
    data['surname'] = this.surname;
    data['phone'] = this.phone;
    data['mail'] = this.mail;
    return data;
  }
}
